docs/source/cn/one_embedding.py

```diff
@@ -0,0 +1,270 @@
+    """加载快照。
+
+        参数：
+            - **snapshot_name** (str) - 快照的名称。快照将从路径 your_configed_persistent_path 下加载。
+    
+        示例：
+
+        .. code-block:: python
+
+            > import oneflow as flow
+            > # 使用由 flow.one_embedding.MultiTableEmbedding 创建的 embedding
+            > embedding.load_snapshot("my_snapshot1")
+            > # 在 your_configed_persistent_path 目录下，加载名为 "my_snapshot1" 的快照。
+        """
+)
+
+reset_docstr(
+    oneflow.one_embedding.MultiTableEmbedding.forward,
+    """MultiTableEmbedding 的前向计算。
+
+        参数：
+            - **ids** (flow.tensor) - 特征 id。
+            - **table_ids** (flow.tensor, optional) - 每个 id 的 table_id 必须与 ids 的形状相同。如果只配置了一个表或者 id 的形状为（batch_size, num_tables），并且每个列的 id 都属于表的 column_id，就无需传递 table_ids，否则，应该传递 tensor_ids。
+            
+        返回：
+            - **flow.tensor** - embedding 查询的结果。
+            
+        """
+)
+
+reset_docstr(
+    oneflow.one_embedding.make_device_mem_store_options,
+    """将 MultiTableEmbedding 的参数 store_options 配置为（词表使用）纯 GPU 存储。
+
+    参数：
+        - **persistent_path** (str, list) - Embedding 词表的持久化存储路径。如果传入一个 str，当前 rank 中的 Embedding 词表将被存储在路径 path/rank_id-num_ranks 下；如果传入一个 list，则列表长度必须等于 rank 的数量，列表中的每个元素代表着对应 rank_id 中的 Embedding 词表的存储路径。
+        - **capacity** (int) - Embedding 词表的总容量。
+        - **size_factor** (int, optional) - embedding_dim 的存储大小因子。如果使用 SGD 优化器且 momentum = 0，其值应为 1；如果 momentum > 0，其值应为 2；如果使用 Adam 优化器，其值应为 3。默认值为 1。
+        - **physical_block_size** (int, optional) - 是扇区大小。默认为 512。
+
+    返回值类型：
+        dict
+
+    查看 :func:`oneflow.one_embedding.make_cached_ssd_store_options` 来获得其他信息。
+    """
+)
+
+reset_docstr(
+    oneflow.one_embedding.make_cached_ssd_store_options,
+    """将 MultiTableEmbedding 的参数 store_options 配置为（词表使用）SSD 存储，且使用 GPU 作为高速缓存。
+
+    参数：
+        - **cache_budget_mb** (int) - 单个 GPU 显存作为缓存的预算，单位为 MB。
+        - **persistent_path** (str, list) - Embedding 词表的持久化存储路径。必须使用高速 SSD 由于训练期间频繁的随机磁盘访问。如果传入一个 str，当前 rank 中的 Embedding 词表将被存储在路径 path/rank_id-num_ranks 下；如果传入一个 list，则列表长度必须等于 rank 的数量，列表中的每个元素代表着对应 rank_id 中的 Embedding 词表的存储路径。
+        - **capacity** (int) - Embedding 词表的总容量
+        - **size_factor** (int, optional) - embedding_dim 的存储大小因子。如果使用 SGD 优化器且 momentum = 0，其值应为 1；如果 momentum > 0，其值应为 2；如果使用 Adam 优化器，其值应为 3。默认值为 1。
+        - **physical_block_size** (int, optional) - 是扇区大小。默认为 512。
+
+    返回值类型：
+        dict
+
+    示例：
+
+    .. code-block:: python
+
+        > import oneflow as flow    
+        > store_options = flow.one_embedding.make_cached_ssd_store_options(
+        >     cache_budget_mb=8192, persistent_path="/your_path_to_ssd", capacity=vocab_size,
+        > )
+        > # pass the store_options to the "store_options" param of flow.one_embedding.MultiTableEmbedding
+        > # ...
+    """
+)
+
+reset_docstr(
+    oneflow.one_embedding.make_cached_host_mem_store_options,
+    """将 MultiTableEmbedding 的参数 store_options 配置为（词表使用）CPU 内存存储，且使用 GPU 作为高速缓存。
```

Comment:
```suggestion
    """令主机使用 GPU 作为 MultiTableEmbedding 的缓存 store_options 参数。
```
https://github.com/Oneflow-Inc/oneflow/blob/c5eca53c68b5a55c82f4c75dfed6b455eaf20d64/python/oneflow/one_embedding.py#L427     没有cpu

Reply:
这里的 cpu 我是根据咱们官网上的[实践指南](https://docs.oneflow.org/master/cookies/one_embedding.html#_1)补充的，我觉得英文文档写的不够详细和明确。

Reply:
这里感觉
"将 MultiTableEmbedding 的参数 store_options 配置为（词表使用）CPU 内存存储，且使用 GPU 作为高速缓存。"
的写法比较准确

Reply:
好的